src/views/settings/Applet/AppletHost/AppletHostList.vue

```diff
@@ -93,7 +97,7 @@ export default {
         hasExport: false,
         hasImport: false,
         onCreate: () => {
-          this.$refs.table.onCreate({ type: 'windows', category: 'host', platform: 'RemoteAppHost' })
+          this.$refs.table.onCreate({ query: appletRouteQuery })
         }
       }
     }
```

Comment:
There is no significant deviation from the given JavaScript code. It appears to be working fine until version 2. In addition, it includes some minor changes like `export default` directive which could potentially break if other parts of your application use the same name. Also ensure that you always update your code according to newer requirements.